users/validators.py

```
@@ -16,3 +17,251 @@
+    """
+    if not value:
+        return value
+
+    # Pattern to match all HTML tags
+    tag_pattern = r'<(/?)(\w+)([^>]*)>'
+
+    # Check all tags are allowed
+    for match in re.finditer(tag_pattern, value):
+        tag_name = match.group(2).lower()
+        if tag_name not in ALLOWED_HTML_TAGS:
+            raise ValidationError(f'Tag not allowed: {tag_name}. Use only <a>, <strong>, <em>, <p>, and <br>')
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)